e2e/cypress/integration/channel/more_channels_spec.js

```diff
@@ -141,6 +141,42 @@ describe('Channels', () => {
         // * Assert that archived channel doesn't show up in LHS list
         cy.get('#publicChannelList').should('not.contain', testChannel.display_name);
     });
+
+    it('MM-T1702 Search works when changing public/archived options in the dropdown', () => {
+        cy.apiAdminLogin();
+        cy.apiUpdateConfig({
+            TeamSettings: {
+                ExperimentalViewArchivedChannels: true,
+            },
+        });
```

Comment:
Please add test data "... some public and private archived channels exist" per test case.